packages/angular/src/generators/component-test/component-test.spec.ts

```diff
@@ -0,0 +1,274 @@
+import { libraryGenerator } from '../library/library';
+import { componentTestGenerator } from './component-test';
+jest.mock('@nrwl/cypress/src/utils/cypress-version');
+describe('Angular Cypress Component Test Generator', () => {
+  let tree: Tree;
+  let mockedAssertMinimumCypressVersion: jest.Mock<
+    ReturnType<typeof assertMinimumCypressVersion>
+  > = assertMinimumCypressVersion as never;
+  beforeEach(() => {
+    tree = createTreeWithEmptyWorkspace();
+    // silence warnings about missing .gitignore file
+    tree.write('.gitignore', '');
+    mockedAssertMinimumCypressVersion.mockReturnValue();
+  });
+
+  it('should handle component w/o inputs', async () => {
+    await libraryGenerator(tree, {
+      name: 'my-lib',
+      unitTestRunner: UnitTestRunner.None,
+      linter: Linter.None,
+    });
+    await componentGenerator(tree, {
+      project: 'my-lib',
+      name: 'my-lib',
+    });
+    componentTestGenerator(tree, {
+      componentName: 'MyLibComponent',
+      componentFileName: './my-lib.component',
+      projectPath: 'libs/my-lib',
+      componentDir: 'src/lib/my-lib',
+    });
+    expect(
+      tree.read('libs/my-lib/src/lib/my-lib/my-lib.component.cy.ts', 'utf-8')
+    ).toEqual(`import { MountConfig, mount } from 'cypress/angular';
+import { MyLibComponent } from './my-lib.component';
+
+describe(MyLibComponent.name, () => {
+  const config: MountConfig<MyLibComponent> = {
+    declarations: [],
+    imports: [],
+    providers: []
+  }
+
+  it('renders', () => {
+     mount(MyLibComponent, config);
+  })
+})
+`);
+  });
+
+  it('should generate a component test', async () => {
+    await libraryGenerator(tree, {
+      name: 'my-lib',
+      unitTestRunner: UnitTestRunner.None,
+      linter: Linter.None,
+    });
+    await componentGenerator(tree, {
+      project: 'my-lib',
+      name: 'my-lib',
+    });
+
+    tree.write(
+      'libs/my-lib/src/lib/my-lib/my-lib.component.ts',
+      `
+import { Component, OnInit, Input } from '@angular/core';
+
+export type ButtonStyle = 'default' | 'primary' | 'accent';
+
+@Component({
+  selector: 'proj-my-lib',
+  templateUrl: './my-lib.component.html',
+  styleUrls: ['./my-lib.component.css']
+})
+export class MyLibComponent implements OnInit {
+  @Input('buttonType') type = 'button';
+  @Input() style: ButtonStyle = 'default';
+  @Input() age?: number;
+  @Input() isOn = false;
+  @Input() message: string | undefined;
+  @Input() anotherProp: any;
+  @Input() anotherNeverProp: never;
+  
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+}`
+    );
+
+    componentTestGenerator(tree, {
+      componentName: 'MyLibComponent',
+      componentFileName: './my-lib.component',
+      projectPath: 'libs/my-lib',
+      componentDir: 'src/lib/my-lib',
+    });
+
+    expect(
+      tree.read('libs/my-lib/src/lib/my-lib/my-lib.component.cy.ts', 'utf-8')
+    ).toEqual(`import { MountConfig, mount } from 'cypress/angular';
```

Comment:
This is fine, but it could be worth using Snapshots instead.

An Inline Snapshot would be perfect here.